enhancements/distributed-tracing/distributed-tracing.md

```diff
@@ -0,0 +1,228 @@
+distributed tracing tracks and observes service requests as they flow through a system by
+collecting data as requests go from one service to another. It's possible, then, to
+pinpoint bugs and bottlenecks or other issues that can impact overall system performance.
+Tracing provides the story of an end-to-end request that is difficult to get otherwise.
+
+The OpenTelemetry Collector component can ingest data in OTLP format and be configured to forward
+and export data to a variety of vendor-specific backends, as well as backends that can ingest data
+in OTLP, Jaeger, or Zipkin formats.
+
+## Motivation
+
+As platforms and applications become more distributed and built on microservices or serverless, tracing
+provides an overall picture of system performance. This visibility reveals service dependencies and how
+one component affects another, things which are difficult to observe otherwise. For example, many OpenShift
+bugs or issues are not contained to a single component. Instead, several teams and component owners often
+work together to solve issues and make system improvements. Distributed tracing aids this by
+tracking events across service boundaries. Furthermore, tracing can shrink the time it takes to diagnose issues,
+giving useful information and pinpointing problems without the need for extra code. Upstream, etcd has been
+instrumented to export gRPC traces. CRI-O is also adding instrumentation. Kubernetes API server added the option
+to enable OpenTelemetry tracing in version 1.22. A KEP is under review and work is underway to instrument kubelet.
+A POC has been created with kube-scheduler. With these components instrumented, it will be possible to view traces with
+CRI-O <-> Kubelet <-> Kube-Apiserver <-> ETCD. At this point, there is much to gain in instrumenting
+other components and extending the OpenTelemetry train to give a complete view of the system.
+
+### Goals
+
+Provide an easy way for OpenShift components to add instrumentation for distributed tracing using OpenTelemetry.
+Adding OpenTelemetry tracing spans requires only a few lines of code. The more components that add tracing,
+the more complete the picture will be for anyone who is debugging or trying to understand cluster performance.
+Also, a vendor-agnostic OpenTelemetry Collector with an operator in the works on OperatorHub can be
+temporarily deployed in times of debugging to turn on tracing, and removed when no longer needed. Any component
+that adds instrumentation should add a switch to turn tracing on. It should be easy to enable
+and disable tracing. If enabled but no backend is detected, there should be no performance hit or trace exporter
+connection errors in component logs.
+
+### Non-Goals
+
+The OpenTelemetry Collector operator for OpenShift will not be part of core OpenShift. Instead, the operator is available
+on the OperatorHub in the OpenShift console, or, can be deployed manually.
```

Comment:
Which team will be maintaining that operator?

Reply:
Aside from Operatorhub there is also a [packaged operator from RH](https://docs.openshift.com/container-platform/4.9/distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.html).